### DISCUSSION TITLE
앞으로 구현 해야하는 거

### DISCUSSION THREAD

**Discussion by rlatldnd1** [YOON-JAEYEONG/Student_Survivor#19] [General]
전공, 교양 스킬 알고리즘
스킬 데이터들 정규화
스킬 선택창에서 스킬 선택 후 적용
보스 제외 적들 세분화 : 스폰 타임, 데미지, 체력
보스 컨셉 구체화 및 알고리즘 설계

게임 시작 전에는?
동아리 활동 구현
스킨 상점?

위에 필요한 모든 도트 이미지 그리기

**Comment by YOON-JAEYEONG** [comment]
+레벨업할 때 스킬 뽑는 확률
+일시정지 UI
+메인화면, 게임화면에서 현재 보유중인 골드 확인할 수 있도록
+부활기능

**Reply by YOON-JAEYEONG** [reply]
도 있어야겠네요 말고 또 있으려나

**Reply by rlatldnd1** [reply]
음🤔🤔

**Reply by rlatldnd1** [reply]
+ 젤 처음에 기본 스킬 선택
+ 방어력 구현
+ 코인도 게임 껐다 켜도 저장되게 만들고
